src/components/FilterTags/FilterTags.js

```diff
@@ -179,7 +186,7 @@ const FilterTags = () => {
             activeFilters.length > 0 &&
             activeFilters.map(activeFilter => (
               <span className="p-filter-tags__active-filter" key={activeFilter}>
-                {activeFilter}
+                {activeFilter.replace(":", ": ")}
```

Comment:
This was another drive-by to remove the space in the filter values to make for easier parsing. The UI maintains the space.